roles/fstat/tasks/main.yml

```diff
@@ -31,3 +31,15 @@
     name: fstat
     enabled: True
     state: started
+
+- name: Configure Cron tab centos
```

Comment:
Ohh I didn't know how centos, I have changed the title to configure crontab simply